source/server/options_impl.cc

```diff
@@ -48,6 +48,9 @@ OptionsImpl::OptionsImpl(int argc, char** argv, const HotRestartVersionCb& hot_r
                                         std::thread::hardware_concurrency(), "uint32_t", cmd);
   TCLAP::ValueArg<std::string> config_path("c", "config-path", "Path to configuration file", false,
                                            "", "string", cmd);
+  TCLAP::ValueArg<std::string> config_json(
+      "", "config-json", "JSON configuration, overrides --config-path", false, "",
```

Comment:
i would probably add the word inline to the beginning of the description `inline JSON configuration`

Reply:
Done.